superset-frontend/src/dashboard/components/SliceHeader/index.tsx

```diff
@@ -239,6 +242,9 @@ const SliceHeader: FC<SliceHeaderProps> = ({
       <div className="header-controls">
         {!editMode && (
           <>
+            {SliceHeaderExtension && (
```

Comment:
Can we add a test for when the extension is provided?

Reply:
Good point, done